app.py

```diff
@@ -1,2 +1,40 @@
-from flask import Flask, request
+from flask import Flask, request, jsonify
 import json
+from flask.views import MethodView
+
+
+app = Flask(__name__)
+with open("restaurants.json", "r") as read_file:
+    data = json.load(read_file)
+
+restaurants = data['restaurants']
+
+
+class RestaurantEndpoint(MethodView):
+    def get(self):
+        return jsonify({'restaurants': restaurants}), 200
```

Comment:
`Restaraunt` endpoint is responsible for the resource `restaurants`.
so imagine you see an endpoint like `GET /api/pets`. `pets` sounds like a `list of pet`.
in this case, it is better to respond with a list rather than a dict that has `restaurants` property that is a list of `restaraunt`

Reply:
fixed